client/src/components/TextForm.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import { Header, Segment, Container, Button, Form } from "semantic-ui-react";
+import languageOptions from "./LanguageOptions";
+
+class TextForm extends Component {
```

Comment:
This component can be called AddDocumentForm instead maybe?

Reply:
Replaced